couchbase/src/api/results.rs

```diff
@@ -285,7 +285,24 @@ pub(crate) struct SubDocField {
 }
 
 #[derive(Debug)]
-pub struct MutateInResult {}
+pub struct MutateInResult {
+    content: Vec<SubDocField>,
+    cas: u64,
+}
+
+impl MutateInResult {
+    pub(crate) fn new(content: Vec<SubDocField>, cas: u64) -> Self {
+        Self { content, cas }
+    }
+
+    pub fn cas(&self) -> u64 {
+        self.cas
+    }
+
+    pub fn exists(&self, index: usize) -> bool {
```

Comment:
An exists method is not defined on the MutateInResult (in SDK 3 APIs), but a `contentAs` is since you can get results from certain operations. I can add the latter, but if you remove exists from this commit it would be good

Reply:
I see. Updated the commit removing `exists` method.